<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionList</class>
 <widget class="QWidget" name="QuestionList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Modalidade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="modalitiesComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Região Anatômica:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="anatomicalRegionComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Tarefa:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="assignmentComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addNewQuestionButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar Nova Questão</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteQuestionButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
